### DISCUSSION TITLE
Regarding certain domains in your list

### DISCUSSION THREAD

**Discussion by McDaisy** [lassekongo83/Frellwits-filter-lists#130] [Show and tell]
Hi,

Great job. Ive been using your list for a while now and it does its job. I was trying to understand how domains are added to the list. How are the addition reviewed etc? And what are the possibilities to remove domains/pull request form the list.

Because I found out some certain domains for example one of them: "uso.svenskamassan.se" is a server used for photos not as ads but as icons and presentations use in many websites for different serious and big exhibitions in Gothenburg. It is a shame that such server is blocked in the list. Of course I know how to disable it for myself but I believe that that specific domain is a mistakte to have in your list.

I have a few more but I dont quite understand the possibilities to discuss this.

Thanks!

BR
McDaisy

**Comment by lassekongo83** [comment]
I'm mostly alone in adding domains to the list, with a few exceptions from people that opens issues sometimes. (So that's where erroneous filters and ads/tracking/annoyances should be reported.) Pull requests are trickier as the main.txt file is checked with FOP.py and then sorted with a custom bash script that includes the headers with date etc.

I check some popular sites more often than others almost every day. Smaller sites I may not have time to check very often, so there can be some occasional misses. Sometimes I find new sites through search engines (using various ad related search strings.) I check for ads, tracking, various unnecessary annoyances that covers content or interrupts the user. (Mostly mailing lists modals, anti-adblock, some eu cookie banners, etc.)

The reason for adding `uso.svenskamassan.se` was probably because of the tracking script `uso.svenskamassan.se/business/external/log2uso.js` found on `bokmassan.se`. And probably because I couldn't find the domain anywhere else on either `svenskamassan.se` or `bokmassan.se` I assumed it was only related to tracking. (But it's fixed now.)

The list quality can only get better if more people report issues in the issue tracker.

**Reply by McDaisy** [reply]
Thanks for quick and prompt reply. I see you added the whole url for the js. Great!

Is this the proper channel to report misses like this?


Thanks!

**Comment by lassekongo83** [comment]
Report issues here: https://github.com/lassekongo83/Frellwits-filter-lists/issues

**Comment by blixten85** [comment]
I have found a false flag. 

js.monitor.azure.com

this one is used when you open quick help tool in windows 11
